### PR TITLE
exchange XSaveWorkaround for ucrtFreadWorkaround

### PR DESCRIPTION
**Pull request by shuffle2**

### CODE REVIEW

File: Source/Core/Common/ucrtFreadWorkaround.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	VirtualProtect(patch_addr, patch_size, old_protect, &old_protect);
+
+	FlushInstructionCache(GetCurrentProcess(), patch_addr, patch_size);
+
+	return true;
+}
+
+int __cdecl EnableucrtFreadWorkaround()
+{
+	// This patches ucrtbase such that fseek will alway
```
Comment (Tilka) [inline Source/Core/Common/ucrtFreadWorkaround.cpp]: Typo.

**Comment by Tilka** [comment]
Is there an issue for this? If not, please describe what the problem looks like for a user.

**Comment by JosJuice** [comment]
Yes, https://bugs.dolphin-emu.org/issues/8930

**Comment by dragonbane0** [comment]
Does this for sure fix the issue in question? Cause the crash is still occurring for me.

**Pull request merged**